styles/style.css

```diff
@@ -0,0 +1,148 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
```

Comment:
I didn't realize you could import! This is a really cool approach to incorporate Google fonts.